<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1611153387718' stop='1611153395097'>
    <name>Calculator</name>
    <title>Calculator</title>
    <test-cases>
        <test-case start='1611153387719' status='passed' stop='1611153392163'>
            <name>Addition</name>
            <title>Addition</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='8fdcdfeb-3bf6-40fc-8d02-98b5e849c875-attachment.png' type='image/png' size='21374'/>
            </attachments>
        </test-case>
        <test-case start='1611153392163' status='failed' stop='1611153395095'>
            <name>Substraction</name>
            <title>Substraction</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='f92c9514-9a18-4ecc-9b7e-4b934dd66891-attachment.png' type='image/png' size='20896'/>
            </attachments>
            <failure>
                <message>Expected '4' to equal '5'.</message>
                <stack-trace>Error: Failed expectation
                    at CalculatorPage.verifyOutput (C:\Protractor_Automation\Tests\CalculatorPage.js:22:38)
                    at UserContext.&lt;anonymous> (C:\Protractor_Automation\Tests\Calculator.js:24:24)
                    at C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
                    at new ManagedPromise (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
                    at ControlFlow.promise (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
                    at schedulerExecute (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
                    at TaskQueue.execute_ (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2974:25</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>